pandas/_libs/tslibs/fields.pyx

```diff
@@ -0,0 +1,534 @@
+# -*- coding: utf-8 -*-
+# cython: profile=False
+# cython: linetrace=False
+# distutils: define_macros=CYTHON_TRACE=0
+# distutils: define_macros=CYTHON_TRACE_NOGIL=0
+"""
+Functions for accessing attributes of Timestamp/datetime64/datetime-like
+objects and arrays
+"""
+
+cimport cython
+from cython cimport Py_ssize_t
+
+import numpy as np
+cimport numpy as np
+from numpy cimport ndarray, int64_t, int32_t, int8_t
+np.import_array()
+
+cdef int64_t NPY_NAT = np.datetime64('NaT').astype(np.int64)
```

Comment:
use util.get_nat(), don't change the incantation

Reply:
Had hoped to avoid dependency; changing.